starlingcaptureapi/file_util.py

```diff
@@ -1,9 +1,11 @@
+from . import config
 from hashlib import sha256
 
 import errno
 import logging
 import os
 import uuid
+import subprocess
```

Comment:
I have noticed these lists are typically sorted in alphabetical order. 

I don't know if that is a "style guide" thing or just a co-incident 

@benhylau ?

Reply:
Alphabetic sort is kinda the default style in Python and most (all?) other languages (although in Python there is the added wrinkle of separating blocks into internal / external imports). We don't have really a "style guide" here, other than we autoformat with a default-configured `black`, which does not sort imports. We can add `isort` as well, if that's easier than thinking about the sorting when you add an import.

Reply:
Personally I don't think we need to go implementing `isorts` or even a `style guide`.

Just something I noticed  working on the stack so i tried to follow it. With more people working on the stack was just asking if its something we should be mindful of.

Reply:
I think we don't need extra tooling for this. Sorting alphabetical is something we can get used to as we review each other more.